frontend/src/visualizations/CardRenderer.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component, PropTypes } from "react";
+
+import ExplicitSize from "metabase/components/ExplicitSize.jsx";
+
+import { CardRenderer } from '../card/card.charting';
+
+import { getSettingsForVisualization, setLatitudeAndLongitude } from "metabase/lib/visualization_settings";
+
+@ExplicitSize
+export default class CardRendererComponent extends Component {
```

Comment:
I find it confusing when a component isn't named the same as its filename.  Can we avoid that just to keep things simple and organized?  It makes reading the code easier.